dat/missions/empire/es_cargo.lua

```diff
@@ -23,14 +23,6 @@ require "numstring.lua"
 
 misn_desc  = _("Official Empire cargo transport to %s in the %s system.")
 
-misn_details = _([[
-Cargo: %s (%s)
-Jumps: %d
-Travel distance: %d
-%s
-Time limit: %s
-]])
```

Comment:
Couldn't this pattern be collapsed into a single function instead of multiples? Something like

```cargo_setDesc( cargo, amount, target_system, deadline, notes )```

Where notes could have piracy risk and something like that? Would allow changing order and everything up easier.

Reply:
I'll give it a try.
These users are all unique and beautiful snowflakes (and, irritatingly, they have a reason for it), so more than half of the arguments will have to be nullable (or the function would have to take an arg table).
I hadn't thought about standardizing the order as a reason to do it anyway; that's a fair point.
And it does get nicer if our cargo_setDesc can recalc the routing info. (Caller also uses that to set the reward.)

Reply:
I'm also going to squash, because the first commit had at least one bug.